docs/source/topics/proc_changing-the-selected-preset.adoc

```diff
@@ -0,0 +1,31 @@
+[id="changing-the-selected-preset_{context}"]
+= Changing the selected preset
+
+[role="_abstract"]
+You can change the container runtime used for the {prod} instance by selecting the desired preset.
+
+On {msw} and {mac}, you can change the selected preset using the system tray or command line interface.
+On Linux, use the command line interface.
+
+[IMPORTANT]
+====
+You cannot change the preset of an existing {prod} instance.
+Preset changes are only applied when a {prod} instance is created.
+To enable preset changes, you must delete the existing instance and start a new one.
+====
+
+.Procedure
+
+* Change the selected preset from the command line:
++
+[subs="+quotes,attributes"]
+----
+$ {bin} config preset _<name>_
```

Comment:
```suggestion
$ {bin} config preset __<name>__
```
Per our SSG, I think that variables need two underline characters to render them as italics. At least that's how I've done it. Does a single underline character work?

Reply:
Ah, good catch! Yes, the single underlines do work, but doubling up on them is actually much more robust and should be the general practice. I just forgot. ;)